apptuit/apptuit_client.py

```diff
@@ -83,13 +105,42 @@ def __init__(self, token, api_endpoint="https://api.apptuit.ai/", debug=False):
             port: Port on which the service is running
 
         """
-        if not token:
-            raise ValueError("Invalid token")
         self.token = token
+        if not self.token:
+            self.token = _get_token_from_environment()
+            if not self.token:
+                raise ValueError("Missing Token, '" + APPTUIT_API_TOKEN + "'"
+                                 "is not available in environment variable or"
+                                 "should pass token as parameter to Apptuit")
```

Comment:
The exception message could be
"Missing Apptuit API token, either pass it as a parameter or set as value of the environment variable APPTUIT_TOKEN"